pkg/envelopes/originator.go

```diff
@@ -12,6 +12,14 @@ type OriginatorEnvelope struct {
 	UnsignedOriginatorEnvelope UnsignedOriginatorEnvelope
 }
 
+func NewOriginatorEnvelopeFromBytes(bytes []byte) (*OriginatorEnvelope, error) {
```

Comment:
This is going to cause conflicts. I wound up implementing the same thing in another PR that is about to be merged in.

Sorry about that.